admin/controllers/school.js

```diff
@@ -42,6 +39,19 @@ const getHome = async (req, res, next) => {
   })
 }
 
+const getScorePercentage = async (pupilId) => {
```

Comment:
shouldn't that be within a service instead of the controller?

Reply:
Also it will need to be tested

Reply:
This is a helper function that I extracted from repeated code within the controller.

The school controller needs a complete overhaul (622 lines 😱), so I would rather it be done as one job. 
This needs to go out today for Pauls demo tomorrow.  
I will create a PBI for the controller refactoring.

Reply:
Indeed but we still to only contain pure controller methods every single helper function should exist to a service imo(like a utilService etc.)
Best to keep a note for the refactoring PBI.